components/policy/resources/templates/policy_definitions/BraveSoftware/BraveRewardsDisabled.yaml

```diff
@@ -0,0 +1,37 @@
+caption: Disable Brave Rewards
+default: null
+desc: |-
+  Disable Brave Rewards in Brave.
+
+  Allows users to support content creators and earn tokens for ads seen in Brave. Folks can use tokens to buy gift cards, exchange for other currencies, and more..
```

Comment:
cc: @Miyayes on the wording here! I grabbed it from https://brave.com/brave-rewards/. This text will show in the Windows group policy editor. It looks like this when it's displayed (this screenshot is for Wallet, but just imagine the text above being show for the Rewards group policy)
![image](https://github.com/user-attachments/assets/a0216fe5-819e-42e8-afe3-5fdc69a3a351)

Reply:
Brave Rewards is a feature that allows users to get paid for ads they see in Brave and send contributions to their favorite websites and content creators.